css/design_system/utils/colors.scss

```diff
@@ -0,0 +1,55 @@
+$colors: 
+  "--s-color-black", 
+  "--s-color-white", 
+  "--s-color-cyan-400",
+  "--s-color-cyan-500",
+  "--s-color-cyan-600",
+  "--s-color-cyan-700",
+  "--s-color-gray-100",
+  "--s-color-gray-200",
+  "--s-color-gray-300",
+  "--s-color-gray-400",
+  "--s-color-gray-500",
+  "--s-color-gray-600",
+  "--s-color-gray-700",
+  "--s-color-gray-800",
+  "--s-color-red-500",
+  "--s-color-red-600",
+  "--s-color-red-700",
+  "--s-color-yellow-500",
+  "--s-color-yellow-600",
+  "--s-color-yellow-700";
+
+@mixin docstring($class, $util_type, $color_type, $text) {
+  /**
+   * @title #{$class}
+   * @category utils
+   * @util_type #{$util_type}
+   * @color_group #{$color_type}
+   *
+   * @example
+   * <div class="#{$class}">#{$text}</div>
+   */
+}
+
+@each $color in $colors {
+  $color-name: str-slice($color, 11);
+  $bg-class-name: "s-bg-color-#{$color-name}";
+  $text-class-name: "s-text-color-#{$color-name}";
+  $border-class-name: "s-border-color-#{$color-name}";
```

Comment:
What do you think about omitting the `-color` prefix here? That seems to be what [Tailwind](https://tailwindcss.com/docs/text-color) does and it's still pretty readable.

I.e. we'd use `.s-text-yellow-700` instead of `.s-text-color-yellow-700`. Slightly more compact but still pretty clear whats happening since the color names are self-explanatory.

Reply:
Yes I like it! I actually meant to add as a question in original PR, if `-color` was necessary. I will remove.